overrides.nix

```diff
@@ -8,6 +8,15 @@ self: super: {
     buildPhase = ":";
   } // args);
 
+  doom-modeline = self.straightBuild {
+    pname = "doom-modeline";
+    propagatedBuildInputs = with self; [
+      all-the-icons
+      compat
+      shrink-path
+    ];
+  };
```

Comment:
For some reason only adding `doom-modeline` to `flake.nix` was not sufficient to fix the issue.